lib/shortener/base.js

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, Compiler Explorer Authors
+// Copyright (c) 2020, Compiler Explorer Authors
```

Comment:
no biggie, I thought we were leaving copyrights alone? I forget how we left it :)